torch/_dynamo/variables/misc.py

```diff
@@ -815,8 +815,12 @@ def call_function(
         from .tensor import NumpyNdarrayVariable
 
         options = VariableTracker.propagate([[self]], [args], [list(kwargs.values())])
-        # lookup method name in torch_np
-        if hasattr(torch_np, self.value.__name__):
+        # lookup method name in torch_np. Things like np.dtype(float) are not supported yet.
+        if self.value.__name__ == "dtype":
+            unimplemented(
+                f"numpy dtype function is not supported yet. Got {self.value}."
```

Comment:
Let's not f string the `value` as that could be a huge array, right?

Reply:
type(self.value)?